pkg/server/server_test.go

```diff
@@ -5,12 +5,16 @@ import (
 	"net/http/httptest"
 	"testing"
 
+	"github.com/luke-py-walker/goyagi/pkg/application"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 func TestNew(t *testing.T) {
-	srv := New()
+	app, err := application.New()
```

Comment:
getting this error `assignment mismatch: 2 variables but 1 values`

@nathankhuu any idea what's happening? I thought it was a language convention...

Reply:
um with my limited knowledge of go lol, i think it's because you're returning only 1 thing? https://github.com/Luke-Py-Walker/goyagi/blob/9faa6741aeeacb03f78af13f8dea7360feec9886/pkg/application/application.go#L17

i think you'll add another thing to return in the next part of goyagi
<img width="688" alt="Screen Shot 2020-03-25 at 2 31 45 PM" src="https://user-images.githubusercontent.com/16202989/77587561-60263f80-6ea5-11ea-825b-90e77f637cef.png">

Reply:
lol I think it was just forgotten in this section... I'll add it